emily/lambda/src/resources/deposits.rs

```diff
@@ -0,0 +1,487 @@
+use std::{collections::HashMap, convert::{From, TryFrom}, fmt::format, str::FromStr};
+
+use aws_sdk_dynamodb::types::AttributeValue;
+use emily::models::{self, DepositBasicInfo, OpStatus};
+
+use crate::errors::EmilyApiError;
+
+// ------------------------------------------------------------------
+// Api resources.
+// ------------------------------------------------------------------
+
+/// Deposit resource.
+#[derive(Clone)]
```

Comment:
For these types of data types it's good to be proactive and implement some common traits (in the spirit of https://rust-lang.github.io/api-guidelines/interoperability.html#types-eagerly-implement-common-traits-c-common-traits)

```suggestion
#[derive(Debug, Clone, PartialEq, Eq, Hash)]
```

Reply:
The same for the rest of the types here.